ledger/api-server-damlonx/src/main/scala/com/daml/ledger/api/server/damlonx/services/DamlOnXSubmissionService.scala

```diff
@@ -44,6 +44,8 @@ import scala.concurrent.{ExecutionContext, ExecutionContextExecutor, Future}
 import scala.util.Try
 import java.util.concurrent.{ExecutorService, Executors}
 
+import com.digitalasset.daml.lf.transaction.Node.GlobalKey
+
```

Comment:
Could you move up this import with the other transaction package imports (line 24) ?